types/css-generator/index.d.ts

```diff
@@ -2,15 +2,12 @@
 // Project: https://github.com/luizbills/css-generator.js
 // Definitions by: Septs <https://github.com/septs>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 4.1
 
 export interface Options {
     indentation?: string;
 }
 
-export class Generator {
-    private constructor(options?: Options);
-
+export abstract class Generator {
```

Comment:
why?

Reply:
the private ctor class requires 4.1. This DT is used to consume a package or the purpose of the DT types
is to be used with package itself? (I cannot see reference in the package to use that types).
By cnverting to abstract we are removing the TS version dependency, so the DT can be used and composed with
types that are not necessarily top-notch. 
So the purpose is interoperability. thx!
But up to review, thx!